db/schema.rb

```diff
@@ -56,6 +57,7 @@
     t.string   "question"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.integer  "order"
```

Comment:
Did you add these changes manually? There should be a migration associated with them but I don't see that here. Also, answer_keywords does not need an order.